packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -217,6 +218,11 @@ class Draggable<T extends Object> extends StatefulWidget {
   /// The data that will be dropped by this draggable.
   final T? data;
 
+  /// The cursor for a mouse pointer when the pointer is over the [feedback].
```

Comment:
Does this mean when the pointer is hovering over the `feedback`, the `child` or both? Or is it specific to when drag is active? Can you add more docs here to explain? Thanks!

Reply:
@Piinks 
But the `feedback` widget is visible only when drag is active 😓. How I can clearly explain this without [empty-pose](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose)?

btw
I would like to say thank you for the episode 53 on the Boring Show 🙏